components/xsd-fu/templates-cpp/OMEXMLMetadata.template

```diff
@@ -1,4 +1,5 @@
 {% def indexes_string(indexes) %}${", ".join(map(lambda x: lang.index_string(x),indexes))}{% end %}\
+{% def indexes_string_dummy(indexes) %}${", ".join(map(lambda x: lang.index_string(x, dummy=True),indexes))}{% end %}\
```

Comment:
As the indexes_string_dummy is not being used in this template any more (had previously been in by mistake by the looks of it), the definition can probably be removed